.gitlab-ci.yml

```
@@ -1,126 +1,21 @@
 stages:
-    - empty
+    - build-test
 
-dummy:
-  stage: empty
+trigger-simnet-for-build-and-test:
+  stage:                     build-test
+  variables: 
+    # The branch or tag name for which project is built. 
+    TRGR_REF:                "${CI_COMMIT_REF_NAME}"
+  image:                     "docker.io/paritytech/tools:latest"
   script:
-      - echo nothing to see here
+    - echo 'TRGR_REF is ${TRGR_REF}'
+    - ./scripts/trigger_pipeline.sh
+        --simnet-version=master
+        --upstream-project=simnet_scripts
+        --upstream-ref="${TRGR_REF}"
```

```suggestion
        --upstream-ref="${CI_COMMIT_REF_NAME}"
```
and you could remove `variables:` part from the job definition.